### PR TITLE
Creating the .json file and also adding the one more dense layer for increasing the accuracy

### PR DESCRIPTION
**Pull request by abhishek-mvs**

### CODE REVIEW

**Comment by ruohoruotsi** [comment]
Hi Abhishek, 

Thanks for your PR, below is my feedback:

- your changes will invalidate the comments at the top of the header which show the summary. Since this repo is meant for learning it would be confusing to not have parity between the comments and code. Please update the comments accordingly.

 - Can you please provide some documentation, results so we can see by how much the extra dense layer helps. It intuitively make sense but would be good to provide proper results and computing hardware as I have done in the README.

Let me know if you have any questions.

**Comment by ruohoruotsi** [comment]
Hi Abhishek,
Thanks for your PR. I gave my feedback above. Let me know if you have questions. 
If you do -not- plan on making any additional changes, I'm not going to merge this changes and just close the PR.

File: lstm_genre_classifier_keras.py [hunk 1]
```diff
@@ -124,4 +125,8 @@
 # Creates a HDF5 file 'lstm_genre_classifier.h5'
 model_filename = "lstm_genre_classifier_lstm.h5"
 print("\nSaving model: " + model_filename)
-model.save(model_filename)
\ No newline at end of file
+model.save(model_filename)
+print("creating .json file....")
```
Comment (abhishek-mvs) [inline lstm_genre_classifier_keras.py:129]: Creates a .json file

**Comment by abhishek-mvs** [comment]
Hi,
Sorry for the late response. In the training process, we need to create the .json file which can be used in predict_example.py. 
And about the dense layer, I recently worked on this project but by using CNN. There I observed that the more the dense layers up to a certain level increase the accuracy. So, present I removed the dense layer but there should be a code for generating the json file. Let me know if there is anything that you want from me.

**Comment by ruohoruotsi** [comment]
This PR still has some problems, but can be merged because the https://github.com/ruohoruotsi/LSTM-Music-Genre-Classification/pull/3 introduced a [purely inference file](https://github.com/ruohoruotsi/LSTM-Music-Genre-Classification/pull/3/commits/efa66dd22e9955d20d5261a8990397ff8f118406#diff-e211610844023bcdd484180e6ae70ed10746f4fb437c8effd540fa485cb96d3e) that requires a JSON model config on init, we should generate after training. There are some consistency and readability issues that I'll fix subsequently.
